recipes-kernel/linux/linux-raspberrypi_4.14.bb

```diff
@@ -1,6 +1,6 @@
 LINUX_VERSION ?= "4.14.22"
 
-SRCREV = "d95cdf3e6a0506e3a728b0c80ae02368779d2768"
```

Comment:
Is now an update and not an added recipe anymore.

Reply:
yes this is an update on top of what was pushed as new recipe for 4.14

Reply:
I think what Andrei means is that the commit message is now confusing.